### PR TITLE
Add Ecosia support & fix Phantom regex

### PR DESCRIPTION
**Pull request by lbarthon**

### CODE REVIEW

File: regexes.yaml [hunk 1]
```diff
@@ -217,7 +217,7 @@ user_agent_parsers:
     family_replacement: 'Twitter'
 
   # Phantom app
-  - regex: 'Mozilla.{1,200}Mobile.{1,100}(Phantom\/ios|android).(\d+)\.(\d+)\.(\d+)'
+  - regex: 'Mozilla.{1,200}Mobile.{1,100}(Phantom\/ios|Phantom\/android).(\d+)\.(\d+)\.(\d+)'
```
Comment (migueldemoura) [inline regexes.yaml:220]: Test?

Comment (lbarthon) [reply regexes.yaml:220]: There is one for it, and it's still working! The ecosia test was failing because of that, and it works properly now too

**Review by migueldemoura — APPROVED** [review]

**Comment by ecotopian** [comment]
Hi @migueldemoura @lbarthon. Is there any specific reason that this is not merged? I work at Ecosia and we would fully appreciate if our browsers would be detected by ua-parser 💚 🌳

**Comment by migueldemoura** [comment]
Hi @ecotopian, I'm afraid we aren't maintainers of this project, it'd be down to those folks to get this reviewed and merged. 😕

**Comment by ecotopian** [comment]
@migueldemoura Gotcha! I appreciate the quick reply 🙏 

@commenthol @elsigh @tobie 
Please apologise the callout, but I'd really like to move this forward. Are there any reasons not to? I am happy to assist from Ecosia side to get the ball rolling again on these changes.

**Comment by elsigh** [comment]
Done! Does anyone in this thread want to take a maintainer role by chance?

**Pull request merged**